clang/test/Analysis/stream-error.c

```diff
@@ -259,14 +283,33 @@ void error_indeterminate_clearerr(void) {
   fclose(F);
 }
 
+void error_indeterminate_fgetc(void) {
+  FILE *F = fopen("file", "r+");
+  if (!F)
+    return;
+  int rc = fseek(F, 0, SEEK_SET);
+  if (rc) {
+    if (feof(F)) {
```

Comment:
First branch of `if` is not needed. But I still think that these tests (with `fgetc` and `fputc`) are not testing different conditions than the other test (the condition that we have a warning for _might be 'indeterminate'_), therefore can be removed.